js/mage/adminhtml/variables.js

```diff
@@ -20,27 +20,28 @@ var Variables = {
     overlayShowEffectOptions: null,
     overlayHideEffectOptions: null,
     insertFunction: 'Variables.insertVariable',
-    init: function(textareaElementId, insertFunction) {
-        if ($(textareaElementId)) {
+
+    init: function (textareaElementId, insertFunction) {
```

Comment:
I am not used to the space after `function`. Is this a thing now?

Reply:
done ;-)